obi_one/endpoints/declared_endpoints.py

```diff
@@ -0,0 +1,87 @@
+from app.config import settings
+from app.dependencies.auth import UserContextDep
+from app.dependencies.entitysdk import get_client
+from app.logger import L
+
+from pydantic import BaseModel, Field
+from typing import Annotated
+from fastapi import Depends
+
+from pathlib import Path
+import io
+import tempfile
+
+import entitysdk.client
+import entitysdk.common
+
+from fastapi import APIRouter
+
+from entitysdk.models.morphology import ReconstructionMorphology
+from neurom import load_morphology
+import neurom
+
+def activate_declared_router(router: APIRouter) -> APIRouter:
+
+    class ReconstructionMorphologyMetricsOutput(BaseModel):
+        soma_radius: Annotated[float, Field(title="soma_radius [µm]", description="The radius of the soma in micrometers.")]
+        soma_surface_area: Annotated[float, Field(title="soma_surface_area [µm^2]", description="The surface area of the soma in square micrometers.")]
+        
+    
+    @router.get("/neurom_metrics/{reconstruction_morphology_id}", summary="NeuroM Metrics", description="Takes a single NeuroM morphology and returns the soma radius and surface area.")
+    async def endpoint(entity_client: Annotated[entitysdk.client.Client, Depends(get_client)], 
```

Comment:
"endpoint" , I think you should pick up another name for this function. like compute_metrics . If there is another endpoint for say eFEL for instance.